ext/process/lib.rs

```diff
@@ -640,19 +671,46 @@ fn spawn_child(
 
   let pid = child.id().expect("Process ID should be set.");
 
+  #[cfg(not(windows))]
+  let stdin_rid = child
+    .stdin
+    .take()
+    .map(|stdin| state.resource_table.add(ChildStdinResource::from(stdin)));
```

Comment:
Yikes, all these conditionals are horrible - I think it's fine for this first pass, but we should address it in a follow up - ideally we'd only have one block for windows and one block for unix